apps/bare-expo/ios/Podfile.lock

```diff
@@ -2186,7 +2186,7 @@ EXTERNAL SOURCES:
     :path: "../../../packages/expo-barcode-scanner/ios"
   EXConstants:
     inhibit_warnings: false
-    :path: "../../../packages/expo-constants/ios"
+    :path: "../../../packages/expo-modules-core/ios"
```

Comment:
How is it possible that `EXConstants` pod resolves to expo-modules-core path? 🤔

Reply:
Because I moved it there:

<img width="780" alt="Screenshot 2024-07-21 at 15 12 47" src="https://github.com/user-attachments/assets/ac6fb9f6-ee47-4b40-bd0b-2dd46d774908">

It's a temporary workaround until it's available in the right place. Like I said, it will probably renamed `ExAppConfig` or something similar.